code/modules/projectiles/projectile/magic.dm

```diff
@@ -152,111 +147,65 @@
-				var/animal = pick("parrot","corgi","crab","pug","cat","mouse","chicken","cow","lizard","chick","fox","butterfly","cak")
-				switch(animal)
-					if("parrot")
-						path = /mob/living/simple_animal/parrot
-					if("corgi")
-						path = /mob/living/simple_animal/pet/dog/corgi
-					if("crab")
-						path = /mob/living/simple_animal/crab
-					if("pug")
-						path = /mob/living/simple_animal/pet/dog/pug
-					if("cat")
-						path = /mob/living/simple_animal/pet/cat
-					if("mouse")
-						path = /mob/living/simple_animal/mouse
-					if("chicken")
-						path = /mob/living/simple_animal/chicken
-					if("cow")
-						path = /mob/living/simple_animal/cow
-					if("lizard")
-						path = /mob/living/simple_animal/hostile/lizard
-					if("fox")
-						path = /mob/living/simple_animal/pet/fox
-					if("butterfly")
-						path = /mob/living/simple_animal/butterfly
-					if("cak")
-						path = /mob/living/simple_animal/pet/cat/cak
-					if("chick")
-						path = /mob/living/simple_animal/chick
-
+			var/path = pick(/mob/living/simple_animal/hostile/carp,
+							/mob/living/simple_animal/hostile/bear,
+							/mob/living/simple_animal/hostile/mushroom,
+							/mob/living/simple_animal/hostile/statue,
+							/mob/living/simple_animal/hostile/retaliate/bat,
+							/mob/living/simple_animal/hostile/retaliate/goat,
+							/mob/living/simple_animal/hostile/killertomato,
+							/mob/living/simple_animal/hostile/poison/giant_spider,
+							/mob/living/simple_animal/hostile/poison/giant_spider/hunter,
+							/mob/living/simple_animal/hostile/blob/blobbernaut/independent,
+							/mob/living/simple_animal/hostile/carp/ranged,
+							/mob/living/simple_animal/hostile/carp/ranged/chaos,
+							/mob/living/simple_animal/hostile/asteroid/basilisk/watcher,
+							/mob/living/simple_animal/hostile/asteroid/goliath/beast,
+							/mob/living/simple_animal/hostile/headcrab,
+							/mob/living/simple_animal/hostile/morph,
+							/mob/living/simple_animal/hostile/stickman,
+							/mob/living/simple_animal/hostile/stickman/dog,
+							/mob/living/simple_animal/hostile/megafauna/dragon/lesser,
+							/mob/living/simple_animal/hostile/gorilla,
+							/mob/living/simple_animal/parrot,
+							/mob/living/simple_animal/pet/dog/corgi,
+							/mob/living/simple_animal/crab,
+							/mob/living/simple_animal/pet/dog/pug,
+							/mob/living/simple_animal/pet/cat,
+							/mob/living/simple_animal/mouse,
+							/mob/living/simple_animal/chicken,
+							/mob/living/simple_animal/cow,
+							/mob/living/simple_animal/hostile/lizard,
+							/mob/living/simple_animal/pet/fox,
+							/mob/living/simple_animal/butterfly,
+							/mob/living/simple_animal/pet/cat/cak,
+							/mob/living/simple_animal/chick)
 			new_mob = new path(M.loc)
 
 		if("humanoid")
-			new_mob = new /mob/living/carbon/human(M.loc)
+			if(prob(50))
+				new_mob = new /mob/living/carbon/human(M.loc)
+			else
+				var/hooman = pick(subtypesof(/mob/living/carbon/human/species))
```

Comment:
not how species work

Reply:
The species subtype does all the hard work for us.

Reply:
god i can't wait to remove human species subtypes, they clog the spawn list when i'm testing so i keep having to scroll down to spawn a human

Reply:
too late now sucker hehe >:)